.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 
 node_js:
   - "12"
-  - "node"
```

Comment:
Why'd you drop this?

Reply:
Oh, that's how the commercial stuff set up. Just habit.